haystack/components/converters/__init__.py

```diff
@@ -15,6 +15,7 @@
     "json": ["JSONConverter"],
     "markdown": ["MarkdownToDocument"],
     "msg": ["MSGToDocument"],
+    "multi_file_converter": ["MultiFileConverter"],
```

Comment:
If I understand correctly, we are deciding to put these SuperComponents into existing categories, instead of creating a separate "SuperComponents" category.
I have no strong arguments against this decision, but I would like to hear other opinions as well. @bilgeyucel @julian-risch

Reply:
This was @julian-risch original decision as mentioned in the PR description here https://github.com/deepset-ai/haystack/pull/9135